core/controllers/base.py

```diff
@@ -70,6 +70,27 @@ def get(self):
         auth_services.establish_auth_session(self.request, self.response)
 
 
+class SessionEndHandler(webapp2.RequestHandler):
+    """Class which handles the creation of a new authentication session."""
+
+    def get(self):
+        """Establishes a new auth session."""
+        auth_services.destroy_auth_session(self.response)
+
+
+class CreateInitialSuperAdmin(webapp2.RequestHandler):
```

Comment:
This doesn't appear to be used--why is it needed?

Reply:
It is a temporary handler that is needed to populate the "first user" on the server. Otherwise, the admin is unable to log in to the admin page after the server-with-Firebase is deployed for the first time because the Firebase association doesn't exist yet.